README.md

```diff
@@ -1,32 +1,72 @@
 This is an experiment to create a Docker container that approximates
-the runtime environment of a cloud.gov/CloudFoundry droplet
-using a Python buildpack on the [cflinuxfs2][] stack.
+the runtime environment of a cloud.gov/CloudFoundry droplet on the
+[cflinuxfs2][] stack.
 
 The Dockerfile is heavily inspired by [sclevine/cflocal][].
 
 ## Quick start
+To build an image for a particular buildpack, you provide  the language
+as a argument to the build script. For example, to build an image for `python`:
 
+```shell
+./build.sh python
 ```
-docker-compose build
-docker-compose run app python
+
+This will build a base image that can be used by your own application
+to replicate the build lifecycle when pushing an application.
+
+### Using in your application
+Once you have
+built this image, you can then add a Dockerfile to your own application
+that looks like:
+
+```Dockerfile
+FROM cloud-gov/python
```

Comment:
Yep, it's the difference between build and run.

Providing the `image` to docker-compose will just run the image as-is. We actually need to do a build. To do that, we have the `build` section of the `docker-compose.yml` which requires a `Dockerfile`.